api-starter/src/functions/products-put.js

```diff
@@ -0,0 +1,31 @@
+// import sample data from shared/product-data
+const data = require('../../shared/product-data');
+
+async function ProductsPut(request, context) {
+    try {
+        // get the body from the request
+        const body = await request.json();
+
+        // create a new product object from the body
+        const product = { name: body.name, id: body.id, description: body.description };
+        
+        // call the updateProduct method from the shared/product-data module
+        const updatedProduct = data.updateProduct(product);
+        
+        return { 
+            status: 200, 
+            body: updatedProduct 
```

Comment:
what was the reasoning for body instead of jsonBody on the handlers - remind me of where we left that conversation?

Reply:
Sorry, missed this one - I've changed to jsonBody